### PR TITLE
[GP-3326] other dependencies

### PR DESCRIPTION
**Pull request by avarsava**
JIRA Ticket: GP-3326

- [N/A] Updates Changelog
- [N/A] Updates developer documentation

### CODE REVIEW

File: plugin-mongo/pom.xml [hunk 1]
```diff
@@ -22,7 +22,7 @@
     <dependency>
         <groupId>org.mongodb</groupId>
         <artifactId>mongodb-driver-sync</artifactId>
-        <version>3.11.0</version>
+        <version>4.8.0-beta0</version>
```
Comment (callunity) [inline plugin-mongo/pom.xml:25]: Is there a more stable version available?

File: plugin-sftp/pom.xml [hunk 2]
```diff
@@ -22,7 +22,7 @@
     <dependency>
       <groupId>com.hierynomus</groupId>
       <artifactId>sshj</artifactId>
-      <version>0.27.0</version>
+      <version>0.34.0</version>
```
Comment (callunity) [inline plugin-sftp/pom.xml:25]: :tada:

File: plugin-mongo/pom.xml (hunk 1 above)
Comment (avarsava) [reply plugin-mongo/pom.xml:25]: Both surprised and not surprised that the maven dependency updater thinks beta versions are acceptable. Thanks for pointing this out

Comment (avarsava) [reply plugin-mongo/pom.xml]: Fixed

**Review by callunity — APPROVED** [review]

**Review by mlaszloffy — APPROVED** [review]

**Pull request merged**